bedrock/products/templatetags/misc.py

```diff
@@ -0,0 +1,70 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import jinja2
+
+from django.conf import settings
+from django_jinja import library
+
+
+def _vpn_product_link(product_url, entrypoint, link_text, class_name=None, optional_parameters=None, optional_attributes=None):
+    separator = '&' if '?' in product_url else '?'
+    href = f'{product_url}{separator}entrypoint={entrypoint}&form_type=button&utm_source={entrypoint}&utm_medium=referral'
+
+    if optional_parameters:
+        params = '&'.join('%s=%s' % (param, val) for param, val in optional_parameters.items())
+        href += f'&{params}'
+
+    css_class = 'js-fxa-cta-link js-fxa-product-button'
+    attrs = ''
+
+    if optional_attributes:
+        attrs += ' '.join('%s="%s"' % (attr, val) for attr, val in optional_attributes.items())
+
+    if class_name:
+        css_class += f' {class_name}'
+
+    markup = (f'<a href="{href}" data-action="{settings.FXA_ENDPOINT}" class="{css_class}" {attrs}>'
+              f'{link_text}'
+              f'</a>')
+
+    return jinja2.Markup(markup)
+
+
+@library.global_function
+@jinja2.contextfunction
+def vpn_sign_in_link(ctx, entrypoint, link_text, class_name=None, optional_parameters=None, optional_attributes=None):
+    """
+    Render a vpn.mozilla.org sign-in link with required params for FxA authentication.
+
+    Examples
+    ========
+
+    In Template
+    -----------
+
+        {{ vpn_sign_in_link(entrypoint='mozilla.org', button_text='Sign In') }}
+    """
+    product_url = 'https://vpn.mozilla.org/oauth/init/'
```

Comment:
Just wanting to confirm that we plan on keeping the `vpn.mozilla.org` URL active for auth/services? I wouldn't want a redirect to break these functions. 

/cc @pmclanahan

Reply:
www.mozilla.org will host only the VPN product landing page and waitlist pages. The rest of the functionality will remain at vpn.mozilla.org (such as authentication, purchasing, downloads)